test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java

```diff
@@ -209,4 +211,17 @@ public void nodeTool(int node, String command, String arguments)
         executeAndPrint(fullCommand);
     }
 
+    public static void removeLiveCluster(String temp_dir)
+    {
+        String fullCommand = "ccm remove" + " --config-dir=" + temp_dir;
+        try
+        {
+            logger.debug("Executing: " + fullCommand);
+            Process p = runtime.exec(fullCommand, null, CASSANDRA_DIR);
```

Comment:
execute() in CCMBridge is harcoded for the ccmDir for that instance whereas fullCommand in removeLiveCluster is executed with the latest entry in tempDir.txt